packages/components/src/components/hds/alert/types.ts

```diff
@@ -18,3 +18,15 @@ export enum HdsAlertColorValues {
   Critical = 'critical',
 }
 export type HdsAlertColors = `${HdsAlertColorValues}`;
+
+export enum HdsAlertTagValues {
```

Comment:
question: considering this file is shared across Alert-related files, should we be more explicit with the naming (e.g. `HdsAlertTitleTagValues`)?